src/zope/app/apidoc/static.py

```diff
@@ -222,10 +235,15 @@ def __init__(self, options):
             if self.base_url[-1] != '/':
                 self.base_url += '/'
         else:
-            assert self.options.ret_kind == 'publisher', self.options.ret_kind
             self.browser = PublisherBrowser
             self.base_url = 'http://localhost/'
 
+            if self.options.ret_kind != 'publisher': # the.package[:the_file.zcml]
+                target = self.options.ret_kind.split(':')
+                self.browser.target_package = target[0]
+                if len(target) == 2 and target[1]:
+                    self.browser.zcml_file = target[1]
```

Comment:
Nitpick: `the.package:foo.zcml:42` should be treated as an error (e.g. by using `foo.zcml:42` as the filename and failing with a file not found error) instead of being treated as `the.package:configure.zcml`.

Easiest to do that by changing the `.split(':')` to `.split(':', 1)`.

Reply:
Good point!